src/test/java/io/debezium/connector/yugabytedb/YugabyteDBDatatypesTest.java

```diff
@@ -215,6 +269,44 @@ public void testRecordConsumption() throws Exception {
                 }).get();
     }
 
+    @Test
```

Comment:
Also unrelated, but it would be good to add test for all unwrap/flatten and transform related configs.
this is to ensure that if anything changes, it is detected early.

Reply:
Maybe we can take it up in another PR specifically meant for this ?

Reply:
Ok, please file a task and track it